pages/org/orgdetailpage.js

```diff
@@ -23,7 +24,8 @@ const blankOrg = {
   website: null,
   facebook: null,
   twitter: null,
-  category: ['vp']
+  category: ['vp'],
+  tags: { data: [] }
```

Comment:
this doesn't look quite right. a blankOrg shouldn't have any tags. 
groups: []  perhaps.
Although likely not used at all.

Reply:
You are right. I changed it to `groups: []`